kf_lib_data_ingest/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.3.0'
```

Comment:
Can this be populated directly from git during build/install instead of hard-coded?

Reply:
What do you mean?

Reply:
I mean that this file should be autogenerated.

Reply:
Sure, but probably in another PR. Autogenerating this file should probably be part of the release making process, bc this file should only be updated when we want to make a new release